mangaki/mangaki/models.py

```diff
@@ -72,16 +72,11 @@ def dpp(self, nb_points):
         """
         sample "nb_points" popular works which are far from each other (using DPP)
         """
-        ratings_matrix = RatingsMatrix(Rating.objects.values_list('user_id',
-                                                                  'work_id',
-                                                                  'choice'))
-        similarity = SimilarityMatrix(ratings_matrix.matrix, nb_components_svd=70)
-        set_item_id_popular = set(self.popular()[:NB_POPULAR_WORKS].values_list('pk', flat=True))
-        items = [ratings_matrix.item_dict[item] for item in ratings_matrix.item_set if item in set_item_id_popular]
-        dpp = MangakiDPP(items, similarity.similarity_matrix)
-        liste_dpp = dpp.sample_k(nb_points)
-        liste_dpp_work_ids = [ratings_matrix.item_dict_inv[element] for element in liste_dpp]
-        return self.filter(id__in=liste_dpp_work_ids)
+        work_ids = self.popular().values_list('id', flat=True)[:100]
+        dpp = MangakiDPP(work_ids)
+        dpp.load_from_algo('svd')
```

Comment:
Could you comment on how long does it takes to load from an algorithm? (it makes it easier to think about performance on the big picture afterwards).